components/ActivityCard.js

```diff
@@ -0,0 +1,66 @@
+import Link from "next/link";
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function ActivityCard({ title, categories, imageUrl }) {
+  return (
+    <StyledCardSection>
+      <StyledImageSection>
+        <Image
+          src={imageUrl}
+          alt={title}
+          style={{ objectFit: "cover" }}
+          sizes="33vw"
+          fill
+        />
+      </StyledImageSection>
+      <StyledDetailsSection>
+        <StyledList>
+          {categories.map((category, index) => (
+            <StyledListItem key={index}>{category}</StyledListItem>
```

Comment:
never ever use the index of the array as the key for the react element. The index is not bound the content (the order can change). The key must always be bound to the content of the element.